tilequeue/command.py

```diff
@@ -1070,6 +1070,84 @@ def tilequeue_dump_tiles_of_interest(cfg, peripherals):
     )
 
 
+def tilequeue_load_tiles_of_interest(cfg, peripherals):
+    """
+    Given a newline-delimited file containing tile coordinates in
+    `zoom/column/row` format, load those tiles into a new Redis SET
+    alongside the existing one, then RENAME the new to the old.
+    """
+    logger = make_logger(cfg, 'load_tiles_of_interest')
+    logger.info('Loading tiles of interest')
+
+    toi_filename = "/Users/iandees/Downloads/tile_requests_unique_noeurope.txt"
+    chunk_size = 10000
+    new_toi_key = cfg.redis_cache_set_key + "-new"
+    old_toi_key = cfg.redis_cache_set_key + "-old"
+
+    def grouper(iterable, n, fillvalue=None):
+        """Collect data into fixed-length chunks or blocks"""
+        args = [iter(iterable)] * n
+        return izip_longest(fillvalue=fillvalue, *args)
```

Comment:
I'm surprised this worked, because it's a non keyword arg after a keyword arg, but I guess the `*args` is special and python is ok with it. I would have expected to have to write: `izip_longest(*args, fillvalue=fillvalue)`.
(Nothing to change here, just thinking out loud) :)